webapp/src/Utils/Scoreboard/Scoreboard.php

```diff
@@ -438,4 +450,28 @@ public function solvedFirst(Team $team, ContestProblem $problem): bool
     {
         return $this->matrix[$team->getTeamid()][$problem->getProbid()]->isFirst;
     }
+
+
+    /**
+     * Determine whether this team has the fastest correct implementation for this problem
+     */
+    public function isFastestImplementation(Team $team, ContestProblem $problem): bool
```

Comment:
Nitpick: let's stick to existing terminology:
```suggestion
    public function isFastestSubmission(Team $team, ContestProblem $problem): bool
```